config/management/src/lib.rs

```diff
@@ -339,58 +392,47 @@ pub mod tests {
         let mut storage = secure_storage(default_secure_backend(namespace.into()));
         storage.reset_and_clear().unwrap();
 
-        let output = verify(namespace).split("KeyNotSet").count();
+        let output = verify(namespace).unwrap().split("KeyNotSet").count();
         assert_eq!(output, 10); // 9 KeyNotSet results in 9 splits
 
         initialize_storage(default_secure_backend(namespace.into()));
 
-        let output = verify(namespace).split("KeyNotSet").count();
+        let output = verify(namespace).unwrap().split("KeyNotSet").count();
         assert_eq!(output, 1); // 0 KeyNotSet results in 1 split
     }
 
     #[test]
-    #[should_panic]
     #[ignore]
-    fn test_owner_key_invalid() {
+    fn test_owner_key() {
         let namespace = "owner_key";
         let mut storage = secure_storage(default_secure_backend(namespace.into()));
         storage.reset_and_clear().unwrap();
-        owner_key(namespace);
-    }
+        owner_key(namespace).unwrap_err();
 
-    #[test]
-    #[ignore]
-    fn test_owner_key_valid() {
-        let namespace = "owner_key";
         let storage = initialize_storage(default_secure_backend(namespace.into()));
         let key = storage
             .get_public_key(constants::OWNER_KEY)
             .unwrap()
             .public_key;
-        let okey = owner_key(namespace);
+        let okey = owner_key(namespace).unwrap();
         assert_eq!(key, okey);
     }
 
+
     #[test]
-    #[should_panic]
     #[ignore]
-    fn test_operator_key_invalid() {
+    fn test_operator_key_valid() {
```

Comment:
nit: call this "test_operator_key" seeing as though it tests both the valid and invalid cases?

Reply:
there's probably some rebase hell going on here, I realized in development that my order of operations wasn't that great. So I went back to the root commit and did some foundational work and rebased on top of it...